ee/src/main/java/org/jboss/as/ee/component/deployers/ComponentInstallProcessor.java

```diff
@@ -68,9 +64,13 @@
 import org.jboss.msc.service.ServiceName;
 import org.jboss.msc.service.ServiceTarget;
 
-import static org.jboss.as.ee.logging.EeLogger.ROOT_LOGGER;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
```

Comment:
Since you foolishly reordered imports you get to fix the existing incorrect order. ;)

Our code style is:

import static blah.blah  (with no wildcards)

import java.blah
import javax.blah

import other.blah

Reply:
I blame IDEA for that. I'll fix this, then try to figure out how to make IDEA behave.

Reply:
Fixed.